_content/articles/religious-relationships-with-the-environment_woodhouse-et-al.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Religious Relationships with the Environment in a Tibetan Rural Community: Interactions and Contrasts with Popular Notions of Indigenous Environmentalism"
+authors:
+  - "Emily Woodhouse"
+  - "Martin A. Mill"
+  - "Philip J. K. McGowan"
+  - "E. J. Milner-Gulland"
+source_url: "https://doi.org/10.1007/s10745-015-9742-4"
+external_url: "https://link.springer.com/article/10.1007/s10745-015-9742-4"
+drive_links:
+  - "https://drive.google.com/file/d/1oIm2MTWlh9LWixXFUdS_l3pQZstIPT-r/view?usp=share_link"
+formats: [pdf]
+course: nature
+tags:
+  - anthropology
+  - environmentalism
+  - tibet
```

Comment:
`tibet` is good here. Since it is also about approaches to the land and place-making in general you may consider adding `places` as well 😊